server/libraries/commons/commons-stone/src/main/java/com/kamyczki/commons/stone/StoneKafkaConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.kamyczki.commons.stone;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class StoneKafkaConstants {
+
```

Comment:
whitespace here too, dont think intended

Reply:
Intended :) but can be removed with only one field